### DISCUSSION TITLE
blog post has incorrect commands

### DISCUSSION THREAD

**Discussion by justin2004** [minio/minio#20281] [Q&A] [answered]
just wanted to note that some of the commands in this blog post
https://blog.min.io/minio-trino-kubernetes/
aren't valid.

e.g.
```
kubectl minio init -n trino
```
that isn't valid subcommand of kubectl. i think the author might have meant:

```
helm install my-minio bitnami/minio --namespace trino
```

**Comment by harshavardhana** [comment, marked as answer]
`kubectl minio init` is correct, just that that blog was valid for v5.x.x series, not for v6.0.0 - since v6.0.0 has moved to Kustomizew only and no more krew plugin.

https://min.io/docs/minio/kubernetes/upstream/operations/installation.html#kustomize-and-kubectl

> ```
> helm install my-minio bitnami/minio --namespace trino
> ```

This is not our project, so please do not use third-party packages. We do not recommend that you use `bitnami/minio` for MinIO.
